### DISCUSSION TITLE
Process/procedures for adding to VM Community Edition

### DISCUSSION THREAD

**Discussion by bdenton** [s390guy/vm370#110] [Q&A]
I have a couple of finished updates and a few others in the works that I would like to propose be included in Community Edition. I am not clear on some elements of packaging and submission:

1. Is there a master list of assigned update numbers? How do I figure out the "###" part of the update name"HRC###DK"????
2. Is there a preferred way to package the files in an update? On my system, I keep all the files together in a .txt file in CMS READCARD format. How do the keepers of the CE releases prefer to receive proposed updates?
3. What sort of consensus/approval/voting process do I need to go through to share my stuff?

cheers,
William Denton

**Comment by g4ugm** [comment]
William,

Well possibly because there isn’t a process. Bob Bolch built the last one and sadly he passed away recently.

Probably time for a discussion on the H390-vm list @ https://groups.io/g/h390-vm/

Dave

 

From: William Denton ***@***.***> 
Sent: Monday, August 26, 2024 12:13 AM
To: s390guy/vm370 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [s390guy/vm370] Process/procedures for adding to VM Community Edition (Discussion #110)

 

I have a couple of finished updates and a few others in the works that I would like to propose be included in Community Edition. I am not clear on some elements of packaging and submission:

1.	Is there a master list of assigned update numbers? How do I figure out the "###" part of the update name"HRC###DK"????
2.	Is there a preferred way to package the files in an update? On my system, I keep all the files together in a .txt file in CMS READCARD format. How do the keepers of the CE releases prefer to receive proposed updates?
3.	What sort of consensus/approval/voting process do I need to go through to share my stuff?

cheers,
William Denton

**Comment by RossPatterson** [comment]
For anyone interested in this discussion, as Dave suggested, William started an email thread on the h390-vm list at https://groups.io/g/h390-vm/topic/108109885#msg6008

**Comment by RossPatterson** [comment]
@rvjansen started a next-release wish-list a few years ago.  Most of his suggestions were process-focused:

1. The updated BREXX and GCCLIB versions, which have stabilized and extended the BREXX implementation for SixPack 1.3-beta
2. An updated logo screen, this one states 1.2
3. An easy way to extend it with DASD, maybe in the form of an EXEC
4. A procedure to build APL\360 from the IBM provided source
